### PR TITLE
Add README landing page

### LINKED ISSUE #13: populate README

### PR DESCRIPTION
**Pull request by davanstrien**
This is a WIP landing page/README for the hackathon. There are a few things that need to be populated in other places before this can be merged.  closes #13

### CODE REVIEW

**Comment by davanstrien** [comment]
@albertvillanova, this is for adding an initial landing page/readme for the repository. I have tried to keep it relatively concise and will focus on adding more detailed guidance to the wiki.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,47 @@
...
+
+## Goals 
+
+We aim to enable easy discovery and programmatic access to these datasets using Hugging Face's  🤗 [Datasets Hub](https://huggingface.co/datasets). As part of this, we want to:
+
+- Identify datasets that would be useful to have more easily accessible
+- Make these datasets available via the [Datasets Hub](https://huggingface.co/datasets)
+- Document these datasets 
+
+## Why are you doing this?
```
Comment (albertvillanova) [inline README.md:17]: Nit
```suggestion
## Why are we doing this? 
```

**Review by albertvillanova — APPROVED** [review]
Awesome!!! :heart:

File: README.md [hunk 2]
```diff
@@ -1,3 +1,47 @@
...
+- 🤗 Making those datasets available via the Hugging Face Hub
+- 📝 Documenting datasets by adding additional metadata and working on the [data cards](https://huggingface.co/docs/hub/datasets-cards) for those datasets. 
+
+## Joining the hackathon 
+
+To join the hackathon, start by introducing yourself on our GitHub discussion board https://github.com/bigscience-workshop/lam/discussions/19. 
+
+Once you have said hi on the discussion boards you should request to join [BigLAM Hugging Face organization](https://huggingface.co/biglam). 
+
+For guidance on adding datasets, please check out the [Wiki](https://github.com/bigscience-workshop/lam/wiki/contribute_data). If you have questions, please check out the [FAQs](https://github.com/bigscience-workshop/lam/wiki/faq)
```
[the remark this answers is not in the document]
Comment (albertvillanova) [reply README.md:43]: Maybe the text should be rephrased as well...

[the remark this answers is not in the document]
Comment (albertvillanova) [reply README.md:43]: For guidance, please check out the Wiki.

[the remark this answers is not in the document]
Comment (albertvillanova) [reply README.md:43]: For the questions, maybe worth adding a new paragraph. Something like:

> If you have questions,
> - first check out the FAQs
> - make your question in the Discussions

[the remark this answers is not in the document]
Comment (albertvillanova) [reply README.md:43]: And put this new paragraph also in the landing page of the Wiki.

**Pull request merged**